aether-odk-module/aether/odk/api/views_collect.py

```diff
@@ -146,7 +147,9 @@ def xform_list(request, *args, **kwargs):
         xforms = xforms.filter(form_id=formID)
 
     host = request.build_absolute_uri(request.path).replace(reverse('xform-list-xml'), '')
-
+    url_info = urlparse(host)
```

Comment:
Maybe add a little comment that says something `ODK Collect will only use either https or port 8443` to explain why this is here...?